ansible_collections/arista/avd/plugins/plugin_utils/schema/avdtojsonschemaconverter.py

```diff
@@ -3,11 +3,53 @@
 # that can be found in the LICENSE file.
 from __future__ import annotations
 
-from ansible_collections.arista.avd.plugins.plugin_utils.schema.avdschema import AvdSchema
-from ansible_collections.arista.avd.plugins.plugin_utils.schema.avdtodocumentationschemaconverter import get_deprecation
 from ansible_collections.arista.avd.plugins.plugin_utils.schema.key_to_display_name import key_to_display_name
 
 
+def get_deprecation(schema: dict) -> tuple[str, str]:
+    """
+    Build deprecation details for documentation if deprecation is set on the schema element.
+
+    This function is also imported into avdtojsonschemaconverter
+
+    Returns
+    -------
+    deprecation_label : str | None
+        If deprecated or removed this is "removed" or "deprecated". Should be added as label
```

Comment:
This is a bit confusing when reading  
```suggestion
        If deprecated or removed this is "deprecated" or "removed" . Should be added as label
```